### ISSUE TITLE
Atom report missing that lint for eslint when doing step by step instruction.

### ISSUE THREAD

**Issue by senpl** [abhijeetNmishra/webpack-step-by-step#5]
Error while running ESLint: Failed to load plugin jsx-a11y: Cannot find module 'eslint-plugin-jsx-a11y'.

**Comment by senpl** [comment]
```
npm install -g install-peerdeps
install-peerdeps --dev eslint-config-airbnb-base
```
this solve issue for me, but still step by step miss that step.
